### ISSUE TITLE
Quote Indication for 11ty-generated Dendron sites

### ISSUE THREAD

**Issue by imalightbulb** [dendronhq/dendron#477]
**Is your feature request related to a problem? Please describe.**
![image](https://user-images.githubusercontent.com/66677431/107560178-ff683880-6c17-11eb-922c-639ba71a41de.png)
no quote indication

**Describe the solution you'd like**
Add quote indication on 11ty-generated Dendron site.

**Describe alternatives you've considered**
add custom css, but it's quite a troublesome

**Additional context**
Thanks!

**Comment by ScriptAutomate** [comment]
I've run into the same problem. For clarification, for others stumbling upon this issue, this is in reference to using `>` in markdown. This generally should apply some kind of modified formatting to the output of the text, as seen here:

```markdown
This is a test of things. _Italic here._ **Bold here.** _**Bold italics here.**_

> Special formatting, such as for **Note:** type messages, or quotes.
```

Which renders as:

This is a test of things. _Italic here._ **Bold here.** _**Bold italics here.**_

> Special formatting, such as for **Note:** type messages, or featuring quotes.

**Comment by kevinslin** [comment]
Better blockquotes are now released with 0.31
